.github/workflows/unittests.yml

```diff
@@ -0,0 +1,17 @@
+name: Unit Tests
+on:
+  push:
+    branches: [ master ]
```

Comment:
I would add `v3.3` also here - or it needs to be a separate yml on that branch?

Reply:
I'm not sure, but I can cherry-pick the commit to v3.3, good idea

Reply:
I wish we could use some regex here, like `v3.\d`

Reply:
Oh, we can use a pattern (not exactly regex):

```
v3.[0-9]+
```